ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.19__amd64-nightly.yaml

```diff
@@ -1710,24 +1686,24 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-disasterrecovery-sanity
     workflow: cucushift-installer-rehearse-azure-ipi-private
-- as: azure-ipi-proxy-workers-rhcos-rhel8-f14
-  cron: 34 2 12,26 * *
+- as: azure-ipi-proxy-f14
```

Comment:
Oh, this reminds me, before because of  rhel worker, we have to place it in the amd64 ci yaml, now I think we can move it to multi ci yaml, one uses amd64 arch, another uses arm64 arch.

Reply:
ok, let me update this